### ISSUE TITLE
No sound when message is recieved or when being called

### ISSUE THREAD

**Issue by ghost** [qTox/qTox#2606]
I have the option in my settings called "Play Sound" for chat ticked, and yet whenever I recieve a message I hear no sound, though I do get this strange output in Terminal some of the time when someone sends me a message:

```
[15:16:40.134] src/audio/audio.cpp:236 : Debug: "Opening audio output none"
[15:16:40.134] src/audio/audio.cpp:318 : Debug: Closing output
```

I have also asked a few other people and some of them say that they hear it some of the time, but most of them say that they don't hear it at all and never have done. And I have certainly never heard this noise.

---

**OS Information:**

```
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 15.10
Release:    15.10
Codename:   wily
Flavour: GNOME
GNOME Version: 3.18
```

**Package Information:**

```
qtox:
  Installed: 268-0ef39aa
  Candidate: 268-0ef39aa
  Version table:
 *** 268-0ef39aa 0
        500 https://pkg.tox.chat/debian/ nightly/release amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by ghost** [comment]
I have also now noticed that I hear no sound when being called.

**Comment by tux3** [comment]
@SeventhSonOfASeventhSon Do you have any audio output selected in the Audio settings?

**Comment by ghost** [comment]
Yes, I do, and although call quality is very bad, I can still hear something so I can verify that my audio settings are configured correctly.

**Comment by ovalseven8** [comment]
@tux3 @antis81 
Just tested. And have two questions:
- When I download latest qTox and create an account, the playback and capture devices are set to 'None' by default. Is that the desired behaviour?
- Please test: Compile or download latest qTox and clean your ~/.config/tox directory. Now, create two accounts (Person A; Person B). Send one message from Person A -> Person B and from Person B -> Person A. You should hear a sound when the message is received. But what's strange: I could hear the sound only when sending the first two messages (from A -> B and B -> A). After that, I can't hear anything. Can you reproduce? There must be a bug. And: Is it desired behaviour that you can hear a sound if playback device is set to 'None'?

**Comment by tux3** [comment]
> When I download latest qTox and create an account, the playback and capture devices are set to 'None' by default. Is that the desired behaviour?

No, this is a bug. I'll try to repro.

> Is it desired behaviour that you can hear a sound if playback device is set to 'None'?

No, we shouldn't play anything is the output device is None. Maybe this one is fixed by #2509

**Comment by ovalseven8** [comment]
> Is it desired behaviour that you can hear a sound if playback device is set to 'None'?
> 
> > No, we shouldn't play anything is the output device is None. Maybe this one is fixed by #2509

As said, it's only for the first messages. Very strange...

**Comment by antis81** [comment]
Should be ok by now -> closing. @ovalseven8 If issue still exists in current master, please reopen.
